src/App.tsx

```diff
@@ -1,35 +1,30 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
-
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
-    <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
+    <div className="flex flex-col items-center justify-center min-h-screen bg-[var(--background-global)] text-[var(--color-text)]">
+      <h1 className="text-4xl font-bold mb-4" style={{ fontFamily: 'var(--font-global)' }}>
+        Breathing Exercise
+      </h1>
+      <button className="px-6 py-2 text-[var(--color-button-text)] bg-[var(--color-button)] rounded-md shadow-md hover:opacity-80">
+        Botón
+      </button>
+
+      <div className="mt-10 flex space-x-4">
+        {/* Círculos de niveles */}
+        <div className="w-30 h-30 rounded-full" style={{ backgroundColor: 'var(--circle-level-1)' }}></div>
```

Comment:
I like you used variable names, so, we can easily adjust it in the future if we decide to change the theme or color palette.